daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1113,6 +1113,13 @@ private[lf] object SBuiltin {
           machine.disclosureTable.contractById.get(SContractId(coid)) match {
             case Some((templateId, arg)) =>
               val coinst = machine.normValue(templateId, arg)
+              val disclosedContract = machine.disclosureTable.disclosedContractById.getOrElse(
+                coid,
+                crash(
```

Comment:
By construction, the disclosure table should have a contract ID to disclosed contract entry for every defined contract key, hence we crash if this is not the case